Casks/p/pro-notes.rb

```diff
@@ -0,0 +1,25 @@
+cask "pro-notes" do
+  version "0.7.6"
+  sha256 "655cd62f286539610c0523fee835ac5f3dcccd74bb9d63e245ee51f5a898e8f5"
+
+  url "https://assets.pronotes.app/downloads/ProNotes-#{version}.zip"
+  name "ProNotes"
+  desc "Apple Notes extension"
+  homepage "https://www.pronotes.app/"
+
+  livecheck do
+    url "https://www.pronotes.app/direct-download"
+    strategy :header_match
+  end
```

Comment:
```suggestion
  livecheck do
    url "https://www.pronotes.app/appcast.xml"
    strategy :sparkle, &:short_version
  end
```

Can you please also add `auto_updates true`

Reply:
How do you find this info? I am just curious.

Reply:
Updated!

Reply:
I used an app called proxyman (`brew install proxyman`) - which allows you to intercept the network calls.